tests/golos.ctrl_tests.cpp

```diff
@@ -244,6 +244,7 @@ BOOST_FIXTURE_TEST_CASE(register_witness, golos_ctrl_tester) try {
     BOOST_CHECK_EQUAL(err.assert_erase_wtnss, ctrl.unreg_witness(_w[0]));
 } FC_LOG_AND_RETHROW()
 
+#if 0
```

Comment:
is test broken?

Reply:
Sorry... On my compiler this tests doesn't compiled. And I have no time to find a reason.